data/data/libvirt/main.tf

```diff
@@ -2,11 +2,18 @@ provider "libvirt" {
   uri = var.libvirt_uri
 }
 
+resource "libvirt_pool" "storage_pool" {
+  name = var.cluster_id
+  type = "dir"
+  path = "/var/lib/libvirt/images/${var.cluster_id}"
```

Comment:
This ends up inside the default pool, if it’s defined following the default configuration; is that what we want? Shouldn’t we use another path, *e.g.* `/var/lib/libvirt/openshift/${var.cluster_id}`?

Reply:
good catch! yeah i think we should not be under the default pool's dir.

Reply:
Updated! I didn't test but that's what the CI is for. :)

Reply:
Tested, works for me:

    DEBUG libvirt_volume.master[0]: Creation complete after 18s [id=/var/lib/libvirt/openshift-images/test1-dj2nv/test1-dj2nv-master-0]

This is with SELinux on too ;-).